exchanges/kraken/kraken_futures.go

```diff
@@ -316,6 +316,12 @@ func (k *Kraken) SendFuturesAuthRequest(ctx context.Context, method, path string
 			"Nonce":   nonce,
 		}
 
+		var futuresURL string
+		futuresURL, err = k.API.Endpoints.GetURL(exchange.RestFutures)
```

Comment:
This is the same as the const in kraken.go, please change and then change the path variable name. 
![image](https://user-images.githubusercontent.com/9391715/199344334-eaf874a2-8fce-463e-b2fc-cd4ea400f89b.png)

Reply:
it looks like `krakenFuturesURL` is not being used anywhere, so I shuffled things around a bit.

Reply:
LGTM! Thanks heaps.